client/system/info_darwin.go

```diff
@@ -58,6 +59,11 @@ func GetInfo(ctx context.Context) *Info {
 	gio.Hostname = extractDeviceName(ctx, systemHostname)
 	gio.WiretrusteeVersion = version.NetbirdVersion()
 	gio.UIVersion = extractUserAgent(ctx)
+	gio.Cloud = detect_cloud.Detect()
```

Comment:
I'd suggest we add the ctx here as well so we can abort depending on the app life cycle. And pass that one to all `Do` calls as well.